packages/warriorjs-cli/templates/README.md.ejs

```diff
@@ -5,6 +5,9 @@
 <% } -%>
 
 ## Level <%- level.number %>
+<% levels.forEach(level => { -%>
+<%- include('levels', { level }); %>
```

Comment:
You need to use the exact [include line below](https://github.com/olistic/warriorjs/pull/199/files#diff-98e59b767e2082dd84aa8e64db82ad47R12) instead of `'levels'` (which you haven't defined yet?). Also, the [header](https://github.com/olistic/warriorjs/pull/199/files#diff-98e59b767e2082dd84aa8e64db82ad47R7) should be rendered inside the loop.